docs/sdk-and-tools/rest-api/multiversx-api.md

```diff
@@ -34,6 +34,14 @@ Checkout information about [pricing](https://blastapi.io/pricing) and API [limit
 
 More details on how to get your private endpoint can be found [here](https://docs.blastapi.io/blast-documentation/tutorials-and-guides/using-blast-to-get-a-blockchain-endpoint-1).
 
+**Kepler**
+
+High-performance infrastructure layer purpose-built for the MultiversX ecosystem.
+
+Low prices, high limits for API, Gateway, Elastic Search, Event Notifier, and many more!
```

Comment:
I would suggest to replace `Low prices` with something more neutral. For example, `Affordable options`

Reply:
Hi @axenteoctavian, thanks for your suggestion.
I updated the description in https://github.com/multiversx/mx-docs/commit/382e3e2c9db4047a23cdf11d278ad728780ddfa7